project_model_to_task/models/project.py

```diff
@@ -75,33 +75,26 @@ def goto_document(self):
 class IrActionActWindows(models.Model):
     _inherit = 'ir.actions.act_window'
 
-    def read(self, cr, uid, ids, fields=None, context=None,
-             load='_classic_read'):
-        if context is None:
-            context = {}
+    def read(self, fields, load='_classic_read'):
 
         def update_context(action):
```

Comment:
It is good practice to declare all the used variables in the funtion signature.
Can use add `self` to it?:  `def update_context(self, action):`